packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,178 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+
+import '../../all-components';
+import type { WaferMapDie, WaferMapColorsScale } from '../types';
+import {
+    WaferMapQuadrant,
+    WaferMapOrientation,
+    WaferMapColorsScaleMode
+} from '../types';
+import {
+    HIGHLIGHTEDVALUESETS,
+    WAFERMAPDIESETS,
+    WAFERMAPCOLORSCALESETS
+} from './sets';
+
+interface WaferMapArgs {
+    quadrant: WaferMapQuadrant;
+    orientation: WaferMapOrientation;
+    maxCharacters: number;
+    dieLabelsHidden: boolean;
+    dieLabelsSuffix: string;
+    colorsScaleMode: WaferMapColorsScaleMode;
+    highlightedValues: number[];
+    dies: WaferMapDie[];
+    colorScale: WaferMapColorsScale;
+}
+
+const metadata: Meta<WaferMapArgs> = {
+    title: 'WaferMap',
+    parameters: {
+        docs: {
+            description: {
+                component: 'A WaferMap description'
+            }
+        },
+        actions: {
+            handles: ['click', 'mouseover']
+        }
+    },
+    render: createUserSelectedThemeStory(html`
+        <nimble-wafer-map
+            id="waferMapEl"
```

Comment:
We should remove the `id`, it should not be reuired to use the wafer-map

Reply:
applied